app/controllers/application_controller.rb

```diff
@@ -14,7 +14,7 @@ def sign_in(user)
   end
 
   def current_user
-    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+    @current_user ||= User.find_by_id(session[:user_id]) if session[:user_id]
```

Comment:
I made this change mostly because I've been dropping and recreating the database a lot when I modify the seeds file when I add tables to the database. If I'm logged in when I drop the database, the next time I open up the app I'd get an ActiveRecord::RecordNotFound exception, which is raised when `#find` doesn't find the record. `#find_by_id` will return nil, and will set `current_user` to nil. It just makes developing a bit easier when I'm playing with populating the database.